reconcile/change_owners/decision.py

```diff
@@ -71,6 +71,9 @@ def apply_decisions_to_changes(
             )
             diff_decisions.append(change_decision)
             for change_type_context in change_decision.coverage:
+                if change_type_context.disabled:
+                    # approvers of a disabled change-type are ignored
+                    continue
```

Comment:
just to be sure, this means that change will not be self-serviced and a app-sre lgtm label will be needed, right ?

Reply:
or we can use it when we learn that a change-type bisbehaves to temporarily remove it from the decision making process until we fixed the issue

Reply:
@geoberle, @patjlm, apologies! I approved the changes.

However do dismiss the review if you decide that the login or changes are not what you want - since the code change itself looks very nice.